massa-ledger-worker/src/ledger_db.rs

```diff
@@ -585,21 +648,22 @@ impl LedgerDB {
         &self,
         addr: &Address,
     ) -> std::collections::BTreeMap<Vec<u8>, Vec<u8>> {
+        let key_prefix = datastore_prefix_from_address(addr);
         let handle = self.db.cf_handle(LEDGER_CF).expect(CF_ERROR);
 
         let mut opt = ReadOptions::default();
-        opt.set_iterate_upper_bound(end_prefix(data_prefix!(addr)).unwrap());
+        opt.set_iterate_upper_bound(end_prefix(&key_prefix).unwrap());
 
         self.db
             .iterator_cf_opt(
                 handle,
                 opt,
-                IteratorMode::From(data_prefix!(addr), Direction::Forward),
+                IteratorMode::From(&key_prefix, Direction::Forward),
             )
             .flatten()
             .map(|(key, data)| {
                 (
-                    key.split_at(ADDRESS_SIZE_BYTES + 1).1.to_vec(),
+                    key.split_at(ADDRESS_SIZE_BYTES + 1).1.to_vec(), // +1 for the type byte, +1 for the length byte
```

Comment:
Change this comment

Reply:
Fixed with https://github.com/massalabs/massa/pull/3455/commits/573ca8243eb7b44bae408426c925c454fc4dfaba